tests/unit/dojson/test_dojson_utils.py

```diff
@@ -127,6 +128,17 @@ def test_create_profile_url_returns_empty_string_when_not_given_an_int():
     assert expected == result
 
 
+def test_force_single_element_returns_first_element_on_a_list():
+    expected = 'foo'
+    result = force_single_element(['foo', 'bar', 'baz'])
+
+    assert expected == result
+
+
+def test_force_single_element_returns_none_on_empty_list():
```

Comment:
If you're writing this test, you can also test that forcing a single element out of a thing that is already a single element returns that element (e.g. `force_single_element('foo') == 'foo'`)